infra/luci/recipes/perfetto.expected/ci_android.json

```diff
@@ -329,10 +330,13 @@
       "latest",
       "-tag",
       "git_revision:",
+      "-metadata",
+      "build_id:8945511751514863184",
```

Comment:
Just for my understanding, where does this come from? Does it need to be the same on android/win/linux?